x-pack/platform/plugins/shared/stack_connectors/server/connector_types/email/index.ts

```diff
@@ -106,6 +107,21 @@ function validateConfig(
     if (config.tenantId == null) {
       throw new Error('[tenantId] is required');
     }
+  } else if (config.service === AdditionalEmailServices.AWS_SES) {
```

Comment:
This validation here is a breaking change. Before this PR I could create the following connector without issues:

```
{
    "name": "test",
    "config": {
        "from": "foo@foo.com",
        "service": "ses",
        "host": "foo.com",
        "port": 123,
        "secure": true,
        "hasAuth": false
    },
    "secrets": {},
    "connector_type_id": ".email"
}
```

Should we validate only if the configuration is set or even not at all? @pmuellr wdyt?

Reply:
But what's the point then in having defaults for this service? Also, in the UI that was not possible.